apps/teams-test-app/src/components/BarCodeAPIs.tsx

```diff
@@ -48,12 +48,32 @@ const RequestBarCodePermission = (): React.ReactElement =>
     },
   });
 
+const WebAPIGetUserMedia = (): React.ReactElement =>
+  ApiWithoutInput({
+    name: 'WebAPIGetUserMedia',
+    title: 'Web API GetUserMedia',
+    onClick: async setResult => {
+      if (navigator.mediaDevices) {
+        navigator.mediaDevices.getUserMedia({ audio: false, video: true }).then(stream => {
```

Comment:
just curious, what kind of stream is this?  is this a readable stream?  (I am just asking, i haven't worked directly with streams a lot, and this is cool)

Reply:
This is video stream

Reply:
The purpose is to show that the media API can be used (and the focus is not so much on the stream, I don't use the stream anywhere)